apolo-cli/src/apolo_cli/formatters/utils.py

```diff
@@ -153,3 +153,38 @@ def yes() -> str:
 
 def no() -> str:
     return "[red]×[/red]"
+
+
+def format_multiple_gpus(entity: Union[_NodePool, Preset], sep: str = "\n") -> str:
+    """
+    Constructs a GPU string from the provided `entity`.
+    Each GPU make will be separated by a `sep` (newline by default), e.g.:
+
+    Nvidia: 10 x tesla
+    AMD: 5 x instinct
+    Intel: 1
+    """
+    gpus = []
+    for gpu_make, gpu_count, gpu_model in (
+        ("Nvidia", entity.nvidia_gpu, entity.nvidia_gpu_model),
+        ("AMD", entity.amd_gpu, entity.amd_gpu_model),
+        ("Intel", entity.intel_gpu, entity.intel_gpu_model),
+    ):
+        if not gpu_count:
+            continue
+        gpus.append(f"{gpu_make}: {format_gpu_string(gpu_count, gpu_model)}")
+
+    return sep.join(gpus)
+
+
+def format_gpu_string(
+    gpu_count: int, gpu_model: Optional[str], sep: str = " x "
```

Comment:
The same, the function is never called with `sep` argument.

Reply:
@asvetlov changed this to a const